etcdctl/ctlv3/command/check.go

```diff
@@ -265,6 +270,30 @@ func newCheckPerfCommand(cmd *cobra.Command, args []string) {
 	}
 }
 
+func attemptCleanup(client *v3.Client) {
+	dctx, dcancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer dcancel()
+	_, err := client.Delete(dctx, checkPerfPrefix, v3.WithPrefix())
+	if err != nil {
+		ExitWithError(ExitError, err)
```

Comment:
I think we should probably just log here that delete failed and not exit.

Reply:
SGTM